src/utils/getFilteredPeople.tsx

```diff
@@ -0,0 +1,64 @@
+import { Person } from '../types';
+
+const ZERO_YEARS = 1500 | 1600 | 1700 | 1800 | 1900 | 2000;
+
+export const getFilteredPeople = (
+  query: string,
+  sex: string,
+  centuries: string[],
+  people: Person[],
+): Person[] => {
+  let filteredPeople = [...people];
+
+  if (query) {
+    filteredPeople = filteredPeople.filter(person => {
+      if (person.name.toLowerCase().includes(query.toLowerCase())) {
+        return person;
+      }
+
+      if (
+        person.motherName &&
+        person.motherName.toLowerCase().includes(query.toLowerCase())
+      ) {
+        return person;
+      }
+
+      if (
+        person.fatherName &&
+        person.fatherName.toLowerCase().includes(query.toLowerCase())
+      ) {
+        return person;
+      }
+
+      return;
+    });
+  }
+
+  if (sex) {
+    filteredPeople = filteredPeople.filter(person => {
+      if (person.sex === sex) {
+        return person;
```

Comment:
Returning `person` within the filter function is unnecessary. You should return `true` or `false` to indicate whether the person should be included in the filtered list.